lib/active_model/serializer/configuration.rb

```diff
@@ -19,6 +19,7 @@ def config.array_serializer
           collection_serializer
         end
 
+        config.default_include_tree = '*'
```

Comment:
How about `default_nesting_level`?
A per-serializer config would raise issues (for instance parent has `'**'` and child has `'*'`).